apps/svelte/src/lib/server/sanity.ts

```diff
@@ -0,0 +1,10 @@
+import { SANITY_API_READ_TOKEN } from '$env/static/private'
+import { client } from '$lib/sanity'
+
+export const serverClient = client.withConfig({
+  token: SANITY_API_READ_TOKEN,
+  stega: {
+    ...client.config().stega,
+    enabled: true,
+  },
```

Comment:
You can use the short-hand for toggling stega:
```suggestion
  stega: true,
```